crates/router/src/core/payments/helpers.rs

```diff
@@ -810,6 +810,30 @@ pub async fn get_customer_from_details<F: Clone>(
     }
 }
 
+/// Get the customer details from customer field if present
+/// or from the individual fields in `PaymentsRequest`
+pub fn get_customer_details_from_request(
```

Comment:
There was a similar case for order_details, backward compatibility was added. But there an error was thrown if data was present in both the fields. Should it be the same case here ?

Reply:
This error has been handled in the validate request.